interface/super/edit_list.php

```diff
@@ -1058,6 +1091,46 @@ function mysubmit() {
                     </select>
                 </div>
 
+                <!--Added filter-->
+                <script type="text/javascript">
+                    function lister() {
+                        var urlFull   = new URL($(location).attr('href'));
+                        var list_from = parseInt($("#list-from").val());
+                        var list_to   = parseInt($("#list-to").val());
+                        var list_id_container = $("#select2-list_id-container").text();
+
+                        if(list_from > list_to || list_from == list_to){
+                            alert("Please enter a enter valid range!");
+                            return false;
+                        }
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_from");
+                            urlFull.searchParams.set("list_from",list_from);
+                        }
+
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_to");
+                            urlFull.searchParams.set("list_to",list_to);
+                        }
+                        if( list_id_container.length > 0 ){
+                            urlFull.searchParams.delete("list_id_container");
+                            urlFull.searchParams.set("list_id_container", list_id_container );
+                        }
+                        window.location.replace(urlFull);
+                    }
+                </script>
+                <?php
+                    $urlFrom = ( isset($_GET["list_from"]) && intval($_GET["list_from"]) >= 0 ? intval($_GET["list_from"]) : (isset($_POST["list_from"]) && intval($_POST["list_from"]) >= 0 ? intval($_POST["list_from"]) : 0) );
+                    $urlTo   = ( isset($_GET["list_to"]) && intval($_GET["list_to"])  >= 0 ? intval($_GET["list_to"]) : (isset($_POST["list_to"]) && intval($_POST["list_to"]) >= 0 ? intval($_POST["list_to"]) : 10) );
+                ?>
+                <div class="blck-filter" style="float: left; margin-top: 5px; margin-left: 10px; border:0px solid red; width: auto; ">
+                    <div id="input-type-from" style="float: left; ">From&nbsp;<input autocomplete="off" id="list-from" value="<?php echo $urlFrom;?>" style = "margin-right: 10px; width: 40px;">
```

Comment:
translate From and html escape $urlFrom:
```
<div id="input-type-from" style="float: left; "><?php echo xlt("From"); ?>&nbsp;<input autocomplete="off" id="list-from" value="<?php echo attr($urlFrom); ?>" style = "margin-right: 10px; width: 40px;">
```

Reply:
Just done.